services/learn-card-network/brain-service/src/helpers/presentation.helpers.ts

```diff
@@ -24,6 +25,14 @@ export const sendPresentation = async (
 
     await createSentPresentationRelationship(from, to, presentationInstance);
 
+    await SendPushNotification({
+        to: to.profileId,
+        message: `${from.displayName} has boosted you!`,
+        title: 'Boost Received',
+        url: `/notifications?claim=true`,
+        actionType: 'redirect',
```

Comment:
I think @gerardopar also needs the uri embedded in this one too, yeah?

Reply:
added